client/app/hosting/database/private-activation/database-private-activation.controller.js

```diff
@@ -1,52 +1,71 @@
-angular.module("App").controller("HostingDatabasePrivateActiveCtrl", ($scope, $rootScope, $stateParams, HostingDatabase, Alerter, Hosting) => {
-    "use strict";
+angular.module("App").controller(
+    "HostingDatabasePrivateActiveCtrl",
+    class HostingDatabasePrivateActiveCtrl {
 
-    $scope.hosting = $scope.currentActionData;
+        constructor ($scope, $rootScope, $stateParams, HostingDatabase, Alerter, PrivateDatabase) {
+            this.$scope = $scope;
+            this.$rootScope = $rootScope;
+            this.$stateParams = $stateParams;
+            this.HostingDatabase = HostingDatabase;
+            this.Alerter = Alerter;
+            this.PrivateDatabase = PrivateDatabase;
 
-    $scope.datas = {
-        versions: []
-    };
-    $scope.loaders = {
-        versions: false
-    };
-    $scope.choice = {
-        ram: $scope.hosting.offerCapabilities.privateDatabases.length === 1 ? $scope.hosting.offerCapabilities.privateDatabases[0] : null,
-        version: null
-    };
+            this.host = this.$scope.currentActionData;
+            this.versions = [];
+            this.loaders = {
+                versions: false
```

Comment:
I guess this is related to data about versions being retrieved from the API so this should be called `versionDataRetrieval` or something of this type